src/main/resources/i18n/messages_fr.properties

```diff
@@ -622,4 +622,5 @@ tournament.status.openForRegistration = Ouvert
 tournament.status.closedForRegistration = Fermé
 tournament.noStartingDate = Pas de date de début
 tournament.noCompletionDate = Pas de date de fin
-settings.notifications.displayNotification = Afficher les notifications
\ No newline at end of file
+settings.notifications.displayNotification = Afficher les notifications
```

Comment:
Please remove all translations ... Translations are only done in the translation tool

Reply:
Got it. Intellij is 'smart' enough to add this stuff automatically.